src/device/libpixy_m4/src/misc.cpp

```diff
@@ -106,24 +106,39 @@ void showError(uint8_t num, uint32_t color, const char *message)
         if (message)
             lpc_printf(message);
     }
- }
+}
 
-uint16_t crc16(const void* const data, const uint32_t size, uint16_t init)
+uint8_t crc8(const void* const data, uint8_t len)
```

Comment:
Just curious... does the MCU have a CRC peripheral that can be used instead?

Reply:
I haven't looked. The effort would be more than it's worth especially when comparing with crc8 lookup table and the small buffer array being used.